omero/sysadmins/server-permissions.rst

```diff
@@ -188,8 +197,15 @@ Permissions tables
 
 The following are the permissions valid for users working on data belonging to
 other group members. These permissions depend on the group permissions and on
-the type of the user performing the action. 
-
+the type of the user performing the action.
+
+**Restricted administrators act as full administrators when using their
+subset of privileges and standard group members for all other actions**. For
```

Comment:
I think it would pay to make two sentences out of this one. I understood it, but afraid thsi is because of my bckgr knowledge.

``...and standard group members for all other actions.`` -> ``For all actions which are not concerned by their privilleges subset, they act as standard group members.``

Reply:
Maybe ``concerned`` -> ``covered``